src/test/app/LoanBroker_test.cpp

```diff
@@ -1021,6 +1021,16 @@ class LoanBroker_test : public beast::unit_test::suite
                 jv[sfLoanBrokerID] = "";
                 env(jv, ter(temINVALID));
             }
+            // preflight: temINVALID (zero broker id)
+            {
+                // needs a flag to distinguish the parsed STTx from the prior test
+                auto jv = del(alice, uint256{}, 1);
```

Comment:
I done led you wrong.

```suggestion
                auto jv = del(alice, uint256{}, tfFullyCanonicalSig);
```